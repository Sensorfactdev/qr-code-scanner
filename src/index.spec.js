import React from 'react';
import renderer from 'react-test-renderer';
import Button from './index';

describe('Button', () => {
  it('should show the given text', () => {
    const text = 'The Text';
    const component = renderer.create(<Button>{text}</Button>).toJSON();
    expect(component).toMatchSnapshot();
  });
});
