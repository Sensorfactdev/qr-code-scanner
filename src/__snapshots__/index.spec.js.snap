// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should show the given text 1`] = `
<button
  className="sc-bdVaJa hWyAij"
  onClick={undefined}
>
  The Text
</button>
`;
